### DISCUSSION TITLE
Texas Hospital Price Transparency

### DISCUSSION THREAD

**Discussion by ACLubin** [CMSgov/hospital-price-transparency#53] [Q&A] [answered]
For the state of Texas, we receive to inquiries regarding price transparency. It seems like Hospital_Price_Transparency@hhs.texas.gov is unaware of our correspondence with hptcompliance@cms.hhs.gov. How can that be resolved?

**Comment by RidenourK** [comment, marked as answer]
@AcLubin. Thank you for your question. 

GitHub discussions are appropriate when you have technical questions or suggestions about the CMS templates or schema themselves, suggestions for additions to the valid values, or want to solicit input from other users of the template about your approach. Please direct questions to CMS about policy to [PriceTransparencyHospitalCharges@cms.hhs.gov](mailto:PriceTransparencyHospitalCharges@cms.hhs.gov). If your hospital is under compliance review, direct your inquiries to [HPTCompliance@cms.hhs.gov](mailto:HPTCompliance@cms.hhs.gov).
